includes/wikia/VariablesExpansions.php

```diff
@@ -513,6 +513,16 @@
     NS_CATEGORY => true
 ];
 
+/**
+ * The Prometheus Pushgateway exists to allow ephemeral and batch jobs to expose their metrics to Prometheus.
+ * Since these kinds of jobs may not exist long enough to be scraped, they can instead push their metrics
+ * to a Pushgateway. The Pushgateway then exposes these metrics to Prometheus.
+ *
+ * @see lib/Wikia/src/Metrics/Collector.php
+ * @var string $wgPrometheusPushgatewayHost
+ */
+$wgPrometheusPushgatewayHost = "pushgateway-prod.$wgWikiaDatacenter.k8s.wikia.net";
```

Comment:
@mixoteric , so where did the idea of being datacenter agnostic go? :) `k8s.wikia.net` correctly resolves to DC specific one in both SJC and RES.